test/data/speedreader/rewriter/pages/news_pages/nypost.com/original.html

```diff
@@ -120,7 +120,7 @@
 				// Check if we have a cookie.
 				if ( GACookie.length ) {
 					// If the cookie value is not empty we have a GA cookie.
-					hasGACookie = GACookie[0].trim().length > 4 ? true : false;
+					hasGACookie = GACookie[0].trim().length > 4;
```

Comment:
Not sure why it's required for test data. In this case it doesn't matter because speedreader ignores any js code.